Compiler/FrontEnd/InstVar.mo

```diff
@@ -1329,7 +1329,7 @@ protected function stripRecordDefaultBindingsFromElement
   output DAE.Element outVar;
   output list<DAE.Element> outEqs;
 algorithm
-  (outVar, outEqs) := match(inVar, inEqs)
+  (outVar, outEqs) := matchcontinue (inVar, inEqs)
```

Comment:
@perost  Instead of using `matchcontinue` you could transform the `true=...` equations into guards of the `match` cases.
We want to gret rid of the `matchcontinue` statements or at least reduce their usage.